standard/lexical-structure.md

```diff
@@ -64,13 +64,16 @@ The productions for *simple_name* ([§11.7.4](expressions.md#1174-simple-names))
 >
 > *end example*
 
```

Comment:
This needs the disable / enable for Markdownlint. (There isn't a better solution yet):

```suggestion
<!-- markdownlint-disable MD028 -->

<!-- markdownlint-enable MD028 -->

```